src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+document.querySelector('.total-population')
+  .innerHTML = String([...document.querySelectorAll('.population')]
+    .map(pop => pop.textContent.split(',').join(''))
```

Comment:
```suggestion
    .map(population => population.textContent.split(',').join(''))
```
do not give shorten names to variables